--- openexr_viewers-1.0.2/exrdisplay/applyCtl.cpp.orig	2007-07-26 04:34:32.000000000 +0200
+++ openexr_viewers-1.0.2/exrdisplay/applyCtl.cpp	2010-10-13 07:09:24.827734339 +0200
@@ -68,6 +68,8 @@
 
 #endif
 
+#include <cstdio>
+#include <cstdlib>
 
 #define WARNING(message) (cerr << "Warning: " << message << endl)
 
--- openexr_viewers-1.0.2/configure.ac.orig	2010-07-23 02:41:37.000000000 +0200
+++ openexr_viewers-1.0.2/configure.ac	2010-10-13 07:21:17.979742720 +0200
@@ -126,6 +126,7 @@
 #include <ImfFrameBuffer.h>
 #include <ImathBox.h>
 #include <vector>
+#include <cstdlib>
     using namespace Ctl;
     using namespace Imf;
     using namespace Imath;
